sharpd/sharp_vty.c

```diff
@@ -425,7 +425,8 @@ DEFPY (install_seg6local_routes,
 	      End_X$seg6l_endx X:X::X:X$seg6l_endx_nh6|\
 	      End_T$seg6l_endt (1-4294967295)$seg6l_endt_table|\
 	      End_DX4$seg6l_enddx4 A.B.C.D$seg6l_enddx4_nh4|\
-	      End_DT6$seg6l_enddt6 (1-4294967295)$seg6l_enddt6_table>\
+	      End_DT6$seg6l_enddt6 (1-4294967295)$seg6l_enddt6_table|\
+	      End_DT6$seg6l_enddt4 (1-4294967295)$seg6l_enddt4_table>\
```

Comment:
This looks wrong. You're using `End_DT6` two times, it should probably be `End_DT4`.